nengo/transforms.py

```diff
@@ -172,9 +174,11 @@ def sample(self, rng=np.random):
             kernel = []
```

Comment:
Can remove this line with the list comprehension

Reply:
Yep, I thought I had removed it but you know how rebasing be